src/Quest.sol

```diff
@@ -74,24 +74,35 @@ contract Quest is ERC721 {
     ////////// Constructor //////////
     /////////////////////////////////
 
+    constructor(
+        string memory _name,
+        string memory _symbol
+    ) ERC721(_name, _symbol) {}
```

Comment:
It wouldn't hurt to call `_disableInitializers()` from the [OZ Initializable contract](https://docs.openzeppelin.com/contracts/4.x/api/proxy#Initializable-_disableInitializers--) in your constructor. (See my comment on line [88](https://github.com/0xZakk/quests-nft/pull/27/files#diff-033480a38b01dcd46dc4992e6fc6f3a217fc40c67602b4f35b110875729168cbR88) about the needed Initializable contract.)

While I don't see any big security needs for this, it's considered best practice to use this to lock implementation contracts that are designed to be called through proxies. In this case it would prevent anyone else from calling the initialize function and renaming your implementation constructs token.